readme.md

```diff
@@ -79,6 +80,28 @@ This is inclusive, thus, when set to `3`, level three headings, are included
 
 Whether to compile list-items tightly (`boolean?`, default: `false`).
 
+###### `options.parents`
+
+Allows headings to be children of certain node types.
+Internally, it uses
+[unist-util-is](https://github.com/syntax-tree/unist-util-is) to check.
+Hence all types that can be passed in as first parameter can be used here,
+including `Function`, `string`, `Object` and `Array.<Test>`.
+Check
+[documentation](https://github.com/syntax-tree/unist-util-is#readme)
+for details.
+(default: `[]`, i.e. only top-level headings are used)
+
+Example:
+
+```json
+{
+  "parents": ["section"]
```

Comment:
```suggestion
  "parents": ["blockquote"]
```

...I think this would be a bit more clear to most people,  as `section` isn’t a standard mdast node type